doc/debugger.rst

```diff
@@ -1388,6 +1402,27 @@ reestablish the link).
 ``msg`` is an optional string elaborating on the reason for the detach.  It may
 or may not be present depending on the nature of detachment.
 
+AppNotify notification (0x07)
+-----------------------------
+
+Format::
+
+    NFY <int: 0x07> [<tval>]* EOM
```

Comment:
Just a small consistency check, this maybe doesn't match other parts of the document, e.g.:

> A request has the format::
>
>    REQ <int: command> <0-N dvalues> EOM

But there are other notations in use too so not a big deal.